.github/workflows/test.yml

```diff
@@ -112,11 +115,22 @@ jobs:
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI }}
+          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI_OIDC }}
+
+      - name: Set up Boto3 with AssumeRoleWithWebIdentity for oidc connectivity
+        run: |
+          export AWS_ROLE_ARN=${{ secrets.AWS_ASSUME_ROLE_CI_OIDC }}
+          export AWS_WEB_IDENTITY_TOKEN_FILE=/tmp/awscreds
+          export AWS_DEFAULT_REGION=ap-southeast-2
```

Comment:
`export`ing these should be unnecessary here, because that just makes the variable available *within* other processes. It looks like all references to these variables in this step are explicit.

Reply:
Arh. Yep. Good point. This was mostly me testing, trying to work out if any other processes were consuming these variables. I have removed and moved them to env as per your other suggestion.